lib/publiccloud/ssh_interactive.pm

```diff
@@ -79,6 +79,8 @@ sub ssh_interactive_leave {
     select_console('tunnel-console') unless ($prev_console =~ /tunnel-console/);
     unset_sshserial_dev();
     set_var('AUTOINST_URL_HOSTNAME', testapi::host_ip());
+    my $test = sub { my $ret; eval { $ret = script_run('true', timeout => 5) == 0 };
+        if ($@) { $ret = 0 }; return $ret };
```

Comment:
```suggestion
    my $test = (script_run('true', timeout => 5) == 0) ? 1 : 0;
```
or `0 : 1` as you need, isn't this way simpler ?

Reply:
no, isn't, `script_call` croaks on timeout :( so its wrapped in eval and `if ($@)` catches that..